README.md

```diff
@@ -264,21 +258,21 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[What API are you getting this data from]**
 
-  - [Answer_1]
+  - [Fake Store API]
 
-- **[Question_2]**
+- **[How can I contribute to this app]**
 
-  - [Answer_2]
+  - [create a new branch push your update, and make a new pull request]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://github.com/samuelmoiwa/react-redux-capstone-project/blob/product_develop/MIT.md) licensed.
```

Comment:
- [ ]  As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.), Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) at the `Acknowledgments` section in your readme file.💯🚀🔥

Reply:
- This issue is well resolved :heavy_check_mark: